Learning_test_automation/Spotkanie8_Zadania/Vitodens343-F_TestsConfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Zawartosc jsona wyglada ok, jak bedziemy go dodawac do SAINTowej solucji, to bedzie trzeba dac inna nazwe pliku, ale to jeszcze pogadamy o tym.